lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -187,9 +188,83 @@
                             })
                         }
                     ]
+                },
+                {
+                    title: gettext('Social Links'),
+                    fields: [
+                        {
+                            view: new AccountSettingsFieldViews.TextFieldView({
+                                model: userAccountModel,
+                                title: gettext('Facebook Link'),
+                                valueAttribute: 'facebook_link',
+                                helpMessage: gettext(
+                                    'Optionally, link to your Facebook profile from your edX learner profile ' +
+                                    'by adding the URL to your personal page or your username, found at ' +
+                                    'facebook.com/[your_username].'
+                                ),
+                                persistChanges: true,
+                                placeholder: gettext('ex: www.facebook.com/edX')
+                            }),
+                        },
+                        {
+                            view: new AccountSettingsFieldViews.TextFieldView({
+                                model: userAccountModel,
+                                title: gettext('Twitter Link'),
+                                valueAttribute: 'twitter_link',
+                                helpMessage: gettext(
+                                    'Optionally, link to your Twitter profile from your edX learner profile ' +
```

Comment:
With the "Optionally" part extracted to the help text under the "Social Media Links" title, this help text can be shortened: "Enter the URL to your Twitter profile, using your Twitter username."